control_center.yml

```
@@ -8,16 +8,15 @@
     - import_role:
         name: confluent.variables
 
+    - name: Populate service facts
+      service_facts:
+
     - name: Check if Control Center Service Running
-      shell: "systemctl show -p SubState {{control_center_service_name}}"
-      changed_when: false
-      # On some SELinux enabled hosts this command will error out, handled in next set_fact task
-      failed_when: false
-      check_mode: false
-      register: substate
+      set_fact:
+        service_state: "{{ ansible_facts.services[control_center_service_name + '.service'].state | default('unknown') }}"
 
     - set_fact:
```

this task does not have a name!
```suggestion
    - name: Set Installation Pattern Variable 
      set_fact:
```

---

also add to other files please.

---

Fixed